File: Hiroshima.API.Project/Hiroshima.Maas.DL/Entities/PassInformationModel/BookedPassInformation.cs
using Hiroshima.Maas.DL.Entities.PGTransactionInformationModel;
using Hiroshima.Maas.DL.Entities.QRCodeModel;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Hiroshima.Maas.DL.Entities.PassInformationModel
{
    public class BookedPassInformation:BaseEntity
    {
        public int PassInformationId { get; set; }
        public int TravellerId { get; set; }
        public string UniqueReferrenceNumber { get; set; }
        public string TransactionNumber { get; set; }
        public string RiderInformation { get; set; }
        public DateTime BookingDate { get; set; }
        [Column(TypeName = "TINYINT(1)")]
        public bool PaymentStatus { get; set; }
        public string PaymentResponse { get; set; }
        public double TotalAmout { get; set; }
        public int Child { get; set; }
        public int Adult { get; set; }
        public string Remark { get; set; }
        public PassInformation PassInformation { get; set; }
        public PGTransactionInformation PGTransactionInformation { get; set; }
        public QRCode QRCode { get; set; }
    }
}

File: Hiroshima.Mobile.Project/src/shared/models/BookedPassInformation.ts
import { IBookedPassInformation } from "../interfaces";
import { PassModel } from ".";

/**
 * @ignore
 */
export class BookedPassInformation implements IBookedPassInformation {
    /**
     * Pass information id
     */
    passInformationId: number;
    /**
     * Traveller device id
     */
    travellerDeviceId: string;
    /**
     * Traveller id
     */
    travellerId?: number;
    /**
     * Unique referrence number
     */
    uniqueReferrenceNumber?: string;
    /**
     * Transaction number
     */
    transactionNumber?: number;
    /**
     * Rider information
     */
    riderInformation?: string;
    /**
     * Booking date
     */
    bookingDate: string;
    /**
     * Payment Status
     */
    paymentStatus?: boolean;
    /**
     * Payment response message
     */
    paymentResponse?: string;
    /**
     * Total amount
     */
    totalAmout: number;
    /**
     * Number of child
     */
    child: number;
    /**
     * Number of adult
     */
    adult: number;
    /**
     * Remark value
     */
    remark?: string;
    /**
     * Pass model information  
     */
    passInformation?: PassModel;
    /**
     * PG transaction model information
     */
    pgTransactionInfo?: any;
    /**
     * QR code model information
     */
    qrCode?: any

}